tests/functions.rs

```diff
@@ -235,3 +235,21 @@ fn works_getlasterror0() {
         assert_eq!(errhandlingapi::GetLastError(), gle())
     }
 }
+
+#[cfg(windows)]
+#[test]
+fn library_open_already_loaded() {
+    use libloading::os::windows::Library;
+    make_helpers();
+
+    // Not loaded yet.
+    assert!(match Library::open_already_loaded(LIBPATH) {
```

Comment:
This seems racy – tests run in parallel and this could very well be executed while some of the other tests has `LIBPATH` loaded.

Reply:
What's the best way to solve this?

Reply:
A couple options here: 
* Locks;
* Some system dll that is not used elsewhere but is known to exist on a windows system;
* Compile a dedicated dll for this test specifically.

Reply:
Changed it to a Rich Edit DLL.